index.js

```diff
@@ -39,6 +41,12 @@ async function getAccountSnapshot(accountNumber) {
 app.use(async (request, response, next) => {
   if (request.path !== '/') {
     const accountNumber = request.path.split('/')[2];
```

Comment:
**question**: could this two throw an index out of bounds error? what is the path going to be ordinarilly?

Reply:
No, JS just returned undefined if you request an index that doesn't exist:
![image](https://github.com/agrc/serverless-print-proxy/assets/1326248/44daac25-0170-4beb-b0a6-fa83eeb6952d)

The path is normally `/v2/<account number>/arcgis/...`

Reply:
so then account number if it wasn't supplied comes in like `/v2/arcgis/` so account number wouldn't be a number then?